src/reducers/systemIntakesReducer.ts

```diff
@@ -1,19 +1,45 @@
-import { STORE_SYSTEM_INTAKES } from '../constants/actions';
-import { SystemIntakesState } from '../types/systemIntake';
+import { SystemIntakesState } from 'types/systemIntake';
+import { prepareSystemIntakeForApp } from 'data/systemIntake';
+import { fetchSystemIntakes } from 'routines/routines';
+import { DateTime } from 'luxon';
 
 const initialState: SystemIntakesState = {
-  systemIntakes: []
+  systemIntakes: [],
+  loaded: false,
+  loadedTimestamp: null,
+  error: null
 };
 
 function systemIntakesReducer(
   state = initialState,
   action: any
 ): SystemIntakesState {
   switch (action.type) {
-    case STORE_SYSTEM_INTAKES:
+    case fetchSystemIntakes.TRIGGER:
       return {
         ...state,
-        systemIntakes: action.systemIntakes
+        loaded: false
```

Comment:
```suggestion
        isLoading: true
```

`fetchSystemIntakes.request()` happens right before the API call is made in the redux saga so that's where the loading state should be set.